src/Components/Button.jsx

```diff
@@ -8,25 +8,15 @@ const styles = {
 	colSpan : 'col-span-2 w-full'
 }
 
-class Button extends React.Component{
-
-	constructor(props){
-		super(props)
-	}
-
-	render(
-		//destructer props
-	  {value,type,handler} = this.props
-
-	){
+function Button({value,type,handler}){
 		return(
```

Comment:
- [ ] Kindly use arrow function instead of conventional functions to follow ES6 😊

Your Function 🔴
```
function button({value,type,handler}){
// your contents
}
```

Arrow Function 🟢
```
const Button = ({value,type,handler}) => {
// your contents
}
```